articles/azure/azs-how-create-scale-set.md

```diff
@@ -0,0 +1,126 @@
+---
+title: How to Create a Scale Set using UKCloud Azure Stack portal | UKCloud Ltd
```

Comment:
Fix capitalisation of title:
How to create a scale set using the UKCloud Azure Stack portal